app/src/components/NavBar.js

```diff
@@ -19,4 +20,95 @@ function NavBar({ isAuthenticated, gcal, setTab, tab }) {
   );
 }
 
+//CALENDAR NAV BAR
+export const CalendarNavBar = ({
+  isAuthenticated,
+  gcal,
+  calView,
+  setCalView,
+  dateNavigation,
+  setDateNavigation,
+}) => {
+  const [gridColumn, setGridColumn] = useState("2 / auto");
+
+  //when user clicks "prev" or "next", tells dateNavigation how much to adjust the dates state by based on calView
```

Comment:
Nice comment, it's direct and clear, makes it a lot easier to understand the rest of this 👌